spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/KafkaStreamsBindingInformationCatalogue.java

```diff
@@ -122,4 +126,19 @@ void addStreamBuilderFactory(StreamsBuilderFactoryBean streamsBuilderFactoryBean
 		return this.streamsBuilderFactoryBeans;
 	}
 
+	public void addThisMap(Map<KStream, ResolvableType> outputBindings) {
+		this.outputBindings = outputBindings;
+	}
+
+	public Map<KStream, ResolvableType> getOutputBindings() {
+		return outputBindings;
+	}
+
+	public void addOutputKStreamResolvable(ResolvableType outboundResolvable) {
+		this.outboundResolvable = outboundResolvable;
+	}
+
+	public ResolvableType getOutboundResolvable() {
```

Comment:
Few lines above you have a method `add. . . ` which is different than `set. . .` since `add` implies some type of collection, and here you are executing `get. . .`. Perhaps a bit more cleanup

Reply:
Also, would be nice if get/set have some matching names

Reply:
done.